requirements/requirements-linux-python3.txt

```diff
@@ -1,2 +1,3 @@
 pyqt5==5.10.1
 lxml==4.6.3
+PyQtWebEngine
```

Comment:
can specify the version here for PyQtWebEngine?
We can freeze it at certain version

Reply:
It's unnecessary. I removed it.